_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -9,6 +9,8 @@ This page describes the Forseti scanners that are available, how they work, and
 why they're important. You can configure Scanner to execute multiple scanners in
 the same run. Learn about [configuring Scanner]({% link _docs/latest/configure/scanner/index.md %}).
 
+---
+
```

Comment:
nit: this file inconsistently quotes yaml files, e.g. it's `a.yaml` and sometimes b.yaml. Please fix everywhere.